examples/getting_started/pt/swarm_script_executor_cifar10.py

```diff
@@ -47,19 +47,25 @@
         executor = ScriptExecutor(task_script_path=train_script)
         job.to(executor, f"site-{i}", gpu=0, tasks=["train", "validate", "submit_model"])
 
-        client_controller = SwarmClientController()
-        job.to(client_controller, f"site-{i}", tasks=["swarm_*"])
-
-        client_controller = CrossSiteEvalClientController()
-        job.to(client_controller, f"site-{i}", tasks=["cse_*"])
-
         # In swarm learning, each client acts also as an aggregator
         aggregator = InTimeAccumulateWeightedAggregator(expected_data_kind=DataKind.WEIGHTS)
         job.to(aggregator, f"site-{i}")
 
         # In swarm learning, each client uses a model persistor and shareable_generator
-        job.to(PTFileModelPersistor(model=Net()), f"site-{i}")
+        persistor = PTFileModelPersistor(model=Net())
+        job.to(persistor, f"site-{i}")
+        shareable_generator = "shareable_generator"
```

Comment:
This should be `shareable_generater=SimpleModelShareableGenerator()`, since we use `job.as_id(shareable_generator)` below. Also we don't need to do `job.to(persistor..)` and `job.to(shareable_generator..)` since they will be already added from the `as_id` calls.

Reply:
removed the un-necessary job.to() calls.